src/Dialogs/PropertiesWindow.vala

```diff
@@ -250,7 +250,7 @@ public class PropertiesWindow : AbstractPropertiesDialog {
         size_value.label = format_size (total_size);
         contains_value.label = get_contains_value (folder_count, file_count);
         update_widgets_state ();
-        update_storage_block_size (total_size, Files.StorageBar.ItemDescription.FILES);
+        storage_levelbar.value = total_size;
```

Comment:
You should not update the storage bar total size here now - it results with the storage bar only showing the selection size, overriding the desired filesystem used + reserved.  This was intended to show a separate block for the selection size (but didnt work)